pkg/acsengine/engine.go

```diff
@@ -487,12 +487,16 @@ func getParameters(cs *api.ContainerService, isClassicMode bool) (map[string]int
 	return parametersMap, nil
 }
 
+func AddValue(m map[string]interface{}, k string, v interface{}) { addValue(m, k, v) }
```

Comment:
Why are we externalizing this function?

(and if we need to, why not just externalize the existing function)